install/advanced/project/index.rst

```diff
@@ -68,15 +74,17 @@ First, we are going to install all the **system packages** needed for the GeoNod
   sudo apt install -y python3-gdal=3.3.2+dfsg-2~focal2 gdal-bin=3.3.2+dfsg-2~focal2 libgdal-dev=3.3.2+dfsg-2~focal2
   sudo apt install -y python3-pip python3-dev python3-virtualenv python3-venv virtualenvwrapper
   sudo apt install -y libxml2 libxml2-dev gettext
-  sudo apt install -y libxslt1-dev libjpeg-dev libpng-dev libpq-dev
+  sudo apt install -y libxslt1-dev libjpeg-dev libpng-dev libpq-dev libmemcached-dev
   sudo apt install -y software-properties-common build-essential
   sudo apt install -y git unzip gcc zlib1g-dev libgeos-dev libproj-dev
   sudo apt install -y sqlite3 spatialite-bin libsqlite3-mod-spatialite
 
   # Install Openjdk
   sudo -i apt update
   sudo apt install openjdk-8-jdk-headless default-jdk-headless -y
-  sudo update-java-alternatives --jre-headless --jre --set java-1.8.0-openjdk-amd64
+
+  # Remember to select the correct java version /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+  sudo update-alternatives --config java
```

Comment:
does this step need user interaction?

Reply:
Yes.

Reply:
Reverted the command back to previous as it works despite the error thrown in terminal.